client/styles/components/_sidebar.scss

```diff
@@ -291,3 +297,7 @@
 .sidebar__file-item--closed .file-item__children {
   display: none;
 }
+
+.file-item__children li:last-child{
```

Comment:
could you pull this color from the theme map so it looks good in the light dark, and high contrast themes?